app/Http/Controllers/UsersController.php

```diff
@@ -435,6 +435,10 @@ public function postBulkEditSave(Request $request)
             if ($request->filled('department_id')) {
                 $update_array['department_id'] = $request->input('department_id');
             }
+            if ($request->filled('city')) {
+                $update_array['city'] = $request->input('city');
+            }
+
```

Comment:
Should this also have Address, State and Country? Or would that make this too wide?

Reply:
I specifically chose not to do that there because I fear people will get that confused with location. It would we weird to bulk edit a bunch of users with an identical home address, wouldn't it?